packages/shared/lib/auxiliary/icon/constants/icon-svg.ts

```diff
@@ -126,7 +132,7 @@ import {
 } from '../svgs'
 import { VERIFICATION_STATUS_OFFICIAL } from '../svgs/verification-status-official.svg'
 
-export const ICON_SVG = {
+export const ICON_SVG: { [key in Icon]: ISvg } = {
```

Comment:
I think we could rename to `ICON_SVG_MAP` to help be more descriptive and could also define a `IconSvgMap` type.